src/pages/MyProfile/MyProfile.js

```diff
@@ -26,7 +28,18 @@ const MyProfile = ({
     setTosOpen(true)
   }
 
-  return <>
+  return <PrimaryLayout
```

Comment:
Here layout component actually helps us, so that we don't have to use fragments :)

Reply:
Also, header stuff will go into layout ;)